sdk/resource/config_test.go

```diff
@@ -23,103 +23,117 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"go.opentelemetry.io/otel"
-	"go.opentelemetry.io/otel/attribute"
 	ottest "go.opentelemetry.io/otel/internal/internaltest"
 	"go.opentelemetry.io/otel/sdk/resource"
 )
 
 const envVar = "OTEL_RESOURCE_ATTRIBUTES"
 
-func TestDefaultConfig(t *testing.T) {
-	store, err := ottest.SetEnvVariables(map[string]string{
-		envVar: "",
-	})
-	require.NoError(t, err)
-	defer func() { require.NoError(t, store.Restore()) }()
+func TestConfig(t *testing.T) {
+	tc := []struct {
+		name      string
+		envars    string
+		detectors []resource.Detector
+
+		resouceValues map[string]string
```

Comment:
`resourceValues` instead of `resouceValues`?

Reply:
Thanks for those.  Look like I didn't even realize that spellcheck wasn't installed in codespaces.